WSUCEG-7140/Minecraft
# Dark clouds issue#28 close#28

Fix [issue#28](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/28)

**Improving Generation and Drawing of Clouds:**
- Separate the `generate_clouds_positions()` function from the `draw_clouds_in_the_sky()` function to ensure each function has a specific role.
- The `generate_clouds_positions()` function determines the coordinates for each block within a cloud.
- The `draw_clouds_in_the_sky()` function takes a list of lists returned from  `generate_clouds_positions()`. Each inner list represents the coordinates of cloud blocks to be drawn in the sky.
- The `draw_clouds_in_the_sky()` function randomly selects the color for each individual cloud (a set of cloud blocks).


**Tests:**
- Add additional tests to verify the ability to locate and draw dark clouds.
- Update existing tests for light clouds to include dark clouds as well.
- Fixed a failed test case to ensure its success, along with the other tests.
